<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1512</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>611</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>输入</string>
   </property>
   <widget class="QLabel" name="Win1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>611</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>210</y>
     <width>581</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>输出</string>
   </property>
   <widget class="QLabel" name="Win2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>581</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>当前状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="state">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="judge">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>分析</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>931</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>天气</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>921</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Sun">
       <property name="text">
        <string>大晴天</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rain">
       <property name="text">
        <string>雨天</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fog">
       <property name="text">
        <string>雾天</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="snow">
       <property name="text">
        <string>雪天</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_ca">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_to_file">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>100</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>从文件里添加</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>30</y>
     <width>281</width>
     <height>761</height>
    </rect>
   </property>
   <property name="title">
    <string>工作记录</string>
   </property>
   <widget class="QTextBrowser" name="show_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>711</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>800</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
